.github/workflows/doc.yml

```diff
@@ -0,0 +1,54 @@
+name: Documentation
+
+on:
+  push:
+    branches: [ $default-branch, ci-sandbox ]
```

Comment:
```suggestion
    branches: [ $default-branch, ci-sandbox ]
  pull_request:
    branches: [ $default-branch ]
```

Reply:
I can remove this following a final approve, right now I demonstrate yaml works using ci-sandbox. Hope you understand.

What I'm trying for is to catch github.ref and push to target folder now attempting to strip `refs/tags`. It'll solve the issue with ci-sandbox.

https://github.community/t/how-to-get-just-the-tag-name/16241

Reply:
This adds a PR trigger; I left the `ci-sandbox` as I assumed you were testing something.